pkg/eventcache/eventcache.go

```diff
@@ -25,7 +25,7 @@ const (
 
 // garbage collection run interval
 const (
-	eventRetryTimer = time.Second * 10
+	eventRetryTimer = time.Second * 1
```

Comment:
`1s` seems reasonable to me, but I wonder if the `10s` timeout was there for some practical reason based on things observed on the field. Is the motivation for this to execute the test faster? In that case, I think we can just have a var for `eventRetryTimer` and modify it for the unit test (and we can even set it lower than `1s`).